scripts/format_translations.py

```diff
@@ -209,9 +187,15 @@ def main():
         data.update(numbers)
         json.dump(data, f, indent=4, sort_keys=True, ensure_ascii=False)
 
-    if args.validate:
-        with open(REFERENCE_LOCALE_PATH, 'r') as f:
-            reference_translation = json.load(f)
+    with open(REFERENCE_LOCALE_PATH, 'r') as f:
+        reference_translation = json.load(f)
+        while True:
+            response = raw_input('Exclude attribute from validation? (y/n)\n')
```

Comment:
As discussed, this may not be the ideal mechanism for excluding attributes. (having to retype the names every time would be tedious and not in the spirit of good automation)

We can always write a specific mechanism now and generalize it later, but we should try not to settle on a mechanism that will be inconvenient for all cases. This comes down to a question about helpful tooling; do you have any suggestions?